proto/net.proto

```diff
@@ -198,12 +232,37 @@ message NetworkFormat {
   }
   optional MovesLeftFormat moves_left = 6;
 
+  enum ActivationFunction {
+    DEFAULT = 0;
```

Comment:
Could call this inherit.

Reply:
My thinking was that inherit is the default action, unless we specify some other default, so this is a bit more flexible. Originally I defined AttentionPolicyActivation, with SELU as default to cover T78-T81 net behavior, but in the end decided that it is better to handle this in the backends with a `NETWORK_SE_WITH_HEADFORMAT` check. Following this, `INHERIT` at this point felt a bit awkward, so I decided to not revert it.

Reply:
based on the comment attached to default activation - we can put a comment in the attention policy section that 'for network type prior to NETWORK_ATTENTIONBODY_WITH_HEADFORMAT' FFN activation in these encoders is SELU - afterwards it follows standard ffn activation function setting.

Reply:
Sounds about right.